inference-engine/src/mkldnn_plugin/nodes/mkldnn_conv_node.h

```diff
@@ -100,10 +74,7 @@ class MKLDNNConvolutionNode : public MKLDNNNode {
                                 MKLDNNMemoryPtr inMem,
                                 MKLDNNMemoryPtr weightMem,
                                 MKLDNNMemoryPtr outMem,
```

Comment:
I would say we do not need memory here. What do you think, may be it is better to request DnnlMemDescPtr here? It should be enough.

Reply:
replaced with `mkldnn::memory::desc`